sim-administration/dw-ssl/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
We should have just one gradle wrapper at the top level project, unless there is a strong need to have one in sub-projects.  So, all the `sim-administration/dw-ssl/gradle` should be deleted.